CONTRIBUTING.md

```diff
@@ -274,6 +274,11 @@ $ cargo test
 
 ## Opening a Pull Request
 
+Before opening a pull request:
+
+1. please check if there is a corresponding issue (_and if not, please make one_), and
+2. assign the issue to yourself by commenting "take" in the issue.
```

Comment:
```suggestion
1. Please check if there is a corresponding issue (_and if not, please make one_).
2. Assign the issue to yourself by commenting "take" in the issue.
3. At the bottom of the PR description, add `Closes #NNNN` where `NNNN` is the
   issue number, so that the issue gets linked to your PR properly. ("Fixes"
   and other keywords that GitHub recognizes are also OK, of course.)
```